src/app/pages/component-sidenav/component-sidenav.scss

```diff
@@ -117,6 +86,14 @@ app-component-sidenav {
   flex: 1 1 auto;
 }
 
+.mat-list-base {
+  padding-top: 0 !important;
```

Comment:
`!important` should be avoided; it's generally considered a major anti-pattern

Reply:
I think the padding is part of the theme so that was the only way I could remove it. (same with `.mat-list-item-content` right below this)

Reply:
You don't need `!important` you just need a selector with higher specificity: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity